_posts/2019-08-04-rspec-described_class.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title:  "`described_class` is Considered Questionable"
```

Comment:
I think the title should be something like "Reconsidering `described_class`" or something along those lines.